kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,4 +1,4 @@
-npm install -g gulp bower
+npm install -g gulp bower yarn
 
 pushd /var/www/miq/vmdb
   bower -F --allow-root install
```

Comment:
@simaishi as I understand it, this file is is for all the things dealing with sui, if that is the case, then yep, bower is no longer needed 🤘